testsuite/tests/tool-ocamldep-strict/classic_error.ml

```diff
@@ -0,0 +1,7 @@
+(* TEST
+modules = "dep.ml a.ml"
+* setup-ocamlc.byte-build-env
+** ocamlc.byte
+commandline = "-depend a.ml doesnotexist.ml"
+*** check-ocamlc.byte-output
```

Comment:
I agree. There's one gotcha in the current implementation though: if we pass as input A B and C, and B does not exist, the output will still contain dependency information for A and C (but with a nonzero exit code). That test kind of characterizes that behaviour so that's why I included it. But yes it can be argued that the output of a failing command is not to be relied upon, so I'm fine with removing the check on the output.